public/app/features/dashboard-scene/conditional-rendering/ConditionalRenderingInterval.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactNode, useState } from 'react';
+
+import { rangeUtil } from '@grafana/data';
+import { SceneComponentProps, sceneGraph } from '@grafana/scenes';
+import { Field, Input, Stack } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+
+import { ConditionHeader } from './ConditionHeader';
+import { ConditionalRenderingBase, ConditionalRenderingBaseState } from './ConditionalRenderingBase';
+import { handleDeleteNonGroupCondition } from './shared';
+
+export type IntervalConditionValue = string;
+interface ConditionalRenderingIntervalState extends ConditionalRenderingBaseState<IntervalConditionValue> {
```

Comment:
nit: this can be simplified to `type ConditionalRenderingIntervalState = ConditionalRenderingBaseState<IntervalConditionValue>`